katharsis-jpa/src/main/java/io/katharsis/jpa/internal/query/AbstractQueryExecutorImpl.java

```diff
@@ -95,8 +95,7 @@ public MetaDataObject getMeta() {
 				entityList.add((T) values[0]);
```

Comment:
source compatiblity is still Java 7, but built with Java 8. Various people asked for it the past few weeks. I think should be ok this way, community growth seems more important than the latest Java 8 features. But we could start using Java 8 for test cases and possibily for non-core parts like new modules.

as disclaimer: I don't like this, but it seems to be the pace of a significant fraction of the java enterprise community. Just this week I saw Katharsis 3 master running perfectly fine in a six year old app server.